docs/development/models/category-group.md

```diff
@@ -32,8 +32,29 @@ lang: php
 
 ## Relationships
 
-- `CategoryField`
-- `Category`
+### `Site`
+
+The MSM Site which the Category Group belongs to.
+
+### `CategoryGroupSettings`
+
+Per-channel settings for this Category Group.
```

Comment:
@intoeetive Is this one correct?

Reply:
Yes it is correct (what makes you asking?)